2001-3000/LIT2814RepCh98.xml

```diff
@@ -1,10 +1,14 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT2814RepCh98">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Malkǝʾ-hymn to Our Lady Mary</title>
+           <title xml:lang="en" corresp="#t1">Salutations to the Virgin Mary</title>
+           <title xml:lang="gez" xml:id="t2">ሰላም፡ ለኪ፡ ደግሚት፡ ሰማይ፡</title>
```

Comment:
I think it should be ዳግሚት፡. It would be the correct form and seems to me like this in the manuscript. Chaîne and the transcription in the manuscript record are probably wrong.

Reply:
Yes, corrected.